build.gradle

```diff
@@ -162,7 +162,7 @@ subprojects {
         findBugsJsr305Version = '3.0.2'
         errorProneVersion = '2.3.2'
         grpcVersion = '1.25.0'
```

Comment:
Just to double check, is this guava version compatible with grpc 1.25?

Reply:
All tests pass.